trino-connector/src/main/java/com/datastrato/gravitino/trino/connector/catalog/CatalogConnectorMetadataAdapter.java

```diff
@@ -108,12 +111,18 @@ public GravitinoSchema createSchema(String schemaName, Map<String, Object> prope
 
   /** Transform gravitino column metadata to trino ColumnMetadata */
   public ColumnMetadata getColumnMetadata(GravitinoColumn column) {
```

Comment:
I suggest adding a properties member variable to GravitinoColumn. This can be handled through each MetadataAdapter's `toGravitinoColumnProperties` and `toTrinoColumnProperties` methods.

Reply:
Fixed